man/systemd.special.xml

```diff
@@ -503,8 +514,11 @@
       <varlistentry>
         <term><filename>sysinit.target</filename></term>
         <listitem>
-          <para>A special target unit covering early boot-up
-          scripts.</para>
+          <para>This target pulls in the services required for system
+          initialization. System services pulled in by this target should
+          be prepared to work without all mount points, and should declare
```

Comment:
It is pulled in, but started concurrently. Perhaps "should be prepared to" is a bit strong though (one can still order a sysinit service after local-fs.target), how about "can be started before local mount points are ready, and so should declare..."?